include/swift/AST/DiagnosticsSema.def

```diff
@@ -4761,9 +4761,19 @@ ERROR(distributed_actor_user_defined_special_property,none,
       "property %0 cannot be defined explicitly, as it conflicts with "
       "distributed actor synthesized stored property",
       (DeclName))
+ERROR(distributed_property_can_only_be_computed_get_only,none,
+      "'distributed' computed property %0 can only be have a 'get' implementation",
+      (DeclName))
+ERROR(distributed_property_cannot_be_static,none,
+      "'distributed' property %0 cannot be 'static'",
+      (DeclName))
+ERROR(distributed_property_can_only_be_computed,none,
+      "static property %0 cannot be 'distributed', because it is not a computed "
+      "get-only property",
+      (DeclName))
 NOTE(distributed_actor_isolated_property,none,
-      "distributed actor state is only available within the actor instance", // TODO: reword in terms of isolation
-      ())
+      "access to this property is only permitted within the distributed actor %0",
+      (DeclName))
```

Comment:
these again try to be more in line with how rest of swift concurrency messages such things

Reply:
Sounds good! "this" is not useful, will do